project/interface.go

```diff
@@ -37,6 +37,7 @@ type APIProject interface {
 	CreateService(name string) (Service, error)
 	AddConfig(name string, config *config.ServiceConfig) error
 	Load(bytes []byte) error
+	ListStoppedContainers(ctx context.Context, services ...string) ([]string, error)
```

Comment:
I would prefer to have a `ListContainer(s)` with some kind of filter or options. This would make it more generic and add ways to do it (I'm even needing this in `libkermit/compose` for example :angel: ).